app/styles/layout/_header.scss

```
@@ -0,0 +1,205 @@
+.header {
+  height: 3.5rem;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  z-index: 1;
+  background: rgba(250, 250, 250, 0.1);
+
+  &-container {
+    height: 100%;
+    @include flex(center, space-between);
+
+    @include responsive(LG) {
+      @include flex(center, flex-end);
+    }
+  }
+  &__menu {
+    position: relative;
+
+    @include responsive(LG) {
+      position: fixed;
+      left: 0;
+      bottom: 0;
+      top: 0;
+      width: 320px;
+      max-width: 100%;
+      background: $secondary-color;
+      padding: 4rem 3rem;
+      transform: translateX(-100%);
+      opacity: 0;
+      transition: all 0.3s linear;
+    }
+  }
+  &__list {
+    @include flex(center, space-between);
+    gap: 0 3rem;
+
+    &-item {
+      position: relative;
+
+      &:not(:first-child)::before {
+        content: "";
+        position: absolute;
+        top: 50%;
+        left: -17.5px;
+        width: 5px;
+        height: 1px;
+        transform: translateY(-50%);
+        background: $gray-color;
+      }
+
+      &:hover a {
+        color: $orange-color;
+        text-decoration: underline;
+      }
+
+      a {
+        color: $gray-color;
+      }
+
+      .active {
+        color: $orange-color;
+        text-decoration: underline;
+      }
+      @include responsive(LG) {
+        font-size: 1.8rem;
+
+        &:not(:first-child)::before {
+          content: unset;
+        }
+
+        .active {
+          color: $title-color;
+          text-decoration: none;
+        }
+        a {
+          color: $title-color;
+        }
+      }
+    }
+
+    @include responsive(LG) {
+      @include flex(flex-start, space-between, nowrap, column);
+      gap: 2rem 0;
+    }
+  }
+  &__search {
+    height: 100%;
+    @include flex(center, space-between);
```

```suggestion
    @include flex(center, space-between);
    height: 100%;
```
include để lên hết trên đầu nhé